### PR TITLE
Migrate to Fluentd v0.14 API

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by repeatedly** [comment]
Sorry for late reply. I need to merge other PR for this patch.
Could you rebase the patch for master?

**Comment by okkez** [comment]
OK I will rebase.

File: lib/fluent/plugin/in_s3.rb [hunk 1]
```diff
@@ -1,115 +1,107 @@
...
-        end
-      end
-    end
-    unless Fluent::Config::ConfigureProxy.method_defined?(:desc)
-      Fluent::Config::ConfigureProxy.class_eval do
-        def desc(description)
-        end
-      end
-    end
+    DEFAULT_PARSE_TYPE = "none"
```
Comment (repeatedly) [inline lib/fluent/plugin/in_s3.rb]: This should be `out_file`: https://github.com/fluent/fluent-plugin-s3/blob/4ed5d9f86ec0b07e41738e6e00398ad88a356b2a/lib/fluent/plugin/out_s3.rb#L97

Comment (repeatedly) [inline lib/fluent/plugin/in_s3.rb]: Ah, sorry. This is input plugin, not output.
My mistake... ;(

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**